include/z64cutscene.h

```diff
@@ -344,6 +344,42 @@ typedef enum CutsceneDestination {
     /* 0x77 */ CS_DEST_ZELDAS_COURTYARD_RECEIVE_LETTER
 } CutsceneDestination;
 
+// values `< 0xFFF0` indicate a "manual" cutscene; can be assigned to
```

Comment:
```suggestion
// values `< 0xFFF0` indicate no cutscene, or a context in which manual cutscenes can be used; can be assigned to
```
Since manual cutscenes are the exception rather than the norm during normal gameplay: a manual cutscene is started with `Cutscene_StartManual`, otherwise if no manual cutscene is started `Cutscene_UpdateManual` (which checks for cutsceneIndex as documented here) is a no-op

Maybe a "see `Cutscene_UpdateManual`" could help understand the PoV from which this is written too